warning: in the working copy of 'C:\Users\hhyyl\Downloads\Stronghold-Protocol-v0.2.1\.scratch\audit-025\base-024\shared\constants.js', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'C:\Users\hhyyl\Downloads\Stronghold-Protocol-v0.2.1\.scratch\audit-025\base-025\shared\constants.js', LF will be replaced by CRLF the next time Git touches it
diff --git "a/C:\\Users\\hhyyl\\Downloads\\Stronghold-Protocol-v0.2.1\\.scratch\\audit-025\\base-024\\shared\\constants.js" "b/C:\\Users\\hhyyl\\Downloads\\Stronghold-Protocol-v0.2.1\\.scratch\\audit-025\\base-025\\shared\\constants.js"
index 8fee9a4..e9865b6 100644
--- "a/C:\\Users\\hhyyl\\Downloads\\Stronghold-Protocol-v0.2.1\\.scratch\\audit-025\\base-024\\shared\\constants.js"
+++ "b/C:\\Users\\hhyyl\\Downloads\\Stronghold-Protocol-v0.2.1\\.scratch\\audit-025\\base-025\\shared\\constants.js"
@@ -8 +8 @@ export const PROTOCOL_VERSION = 1;
-export const APP_VERSION = '0.2.4';
+export const APP_VERSION = '0.2.5';
